packages/components/src/components/functional/CurrentAccountsList/CurrentAccountsList.tsx

```diff
@@ -127,10 +128,9 @@ const CurrentAccountsMain = ({
   useEffect(() => {
     if (!isWebComponent && bankId) {
       const bank = banks?.find((bank) => bank.bankID === bankId)
-
       if (bank) {
         makeToast('success', `Your ${bank.bankName} connection is ready!`)
-        redirect(getRoute('/home/current-accounts'))
+        window.location.href = getRoute('/home/current-accounts')
```

Comment:
Could we use window.location.href exclusively for web components? I'm thinking React Router Dom might offer a better approach – like replacing the page content instead of reloading everything, including all the JS libraries

Something like 
```suggestion
       if (isWebComponent) {
              window.location.href = getRoute('/home/current-accounts')
       } else {
              redirect(getRoute('/home/current-accounts'))
       }
```

Reply:
Yeah we could do but I don't think this is relevant for web components as we don't support connected accounts. If we wanted to use react router it would have to be `navigate` instead of `redirect`. Redirect doesn't do anything in this context. Was there a problem using the hook `useNavigate` when building web components?

Reply:
Since this is wrapped in `if (!isWebComponent && bankId) {` could I switch it back to using `navigate`

Reply:
Ahh, I see what you mean – `navigate` would work, but you're right, we can't use the `useNavigate` hook because it would break the web components. Even wrapping it in a `!isWebComponent` conditional won't help, since the hook loads when the component does, causing it to break right from the start. 😞

Reply:
Ah grand. I thought that was the case.👍